Formula/verapdf-apps.rb

```diff
@@ -0,0 +1,35 @@
+class VerapdfApps < Formula
+  desc "Open-source industry-supported PDF/A validation"
+  homepage "https://verapdf.org/home/"
+  url "https://github.com/veraPDF/veraPDF-apps/archive/refs/tags/v1.20.3.tar.gz"
+  sha256 "18288947ec21c5267512dc8d53582b7268d3124f5c3eda11e17f361b81b281c2"
+  license any_of: ["GPL-3.0-or-later", "MPL-2.0"]
+  head "https://github.com/veraPDF/veraPDF-apps.git", branch: "integration"
+
+  # Even-numbered minor versions represent stable releases.
+  livecheck do
+    url :stable
+    regex(/^v?(\d+\.\d*[02468](?:\.\d+)*)$/i)
+  end
+
+  depends_on "maven" => :build
+  depends_on "openjdk"
+
+  def install
+    system "mvn", "clean", "install"
+
+    installer_file = Pathname.glob("installer/target/verapdf-izpack-installer-*.jar").first
+    system "java", "-DINSTALL_PATH=#{libexec}", "-jar", installer_file, "-options-system"
+
+    bin.install libexec/"verapdf", libexec/"verapdf-gui"
+    bin.env_script_all_files libexec, Language::Java.overridable_java_home_env
+    prefix.install "tests"
+  end
+
+  test do
+    ohai "URL", :stable
+    # with_env(VERAPDF: "#{bin}/verapdf", NO_CD: "1") do
+    #   system prefix/"tests/exit-status.sh"
+    # end
+  end
```

Comment:
Did this need changing?

Reply:
Yes, sorry, I forgot to revert after briefly testing something.